tests/utils/misc/misc.go

```diff
@@ -48,18 +48,13 @@ func WaitForExpectedState(fn getVMPowerStatus, vmName, expectedState string) boo
 	log.Printf("Timed out to poll status\n")
 	return false
 }
-// LogTestStart - Print a start log with given test name and
-// current time stamp
+
+// LogTestStart - Print a start log with given test group and test case name
 func LogTestStart(testGroup string, testName string) {
-	log.Printf("START:%s %s %s", testGroup, testName, curTime())
+	log.Printf("START: %s.%s", testGroup, testName)
```

Comment:
Please retain timestamp, when debugging we know exactly when a test started and when it finished and hence correlate other logs (daemon, plugin). It also lets us know which tests are taking time. Hence it was defined this way.